run.py

```diff
@@ -44,11 +46,36 @@ def parse_args():
     return args
 
 
+def parse_zuliprc(zuliprc_str):
+    zuliprc_path = path.expanduser(zuliprc_str)
+    if not path.exists(zuliprc_path):
+        sys.exit("Error: Cannot find {}".format(zuliprc_path))
+
+    zuliprc = configparser.ConfigParser()
+    zuliprc.read(zuliprc_path)
+
+    # default settings
+    settings = {'theme': 'default'}
+
+    if 'zterm' in zuliprc:
+        if 'theme' in zuliprc['zterm']:
+            settings['theme'] = zuliprc['zterm']['theme']
+
+    return settings
+
+
 def main():
     """
     Launch Zulip Terminal.
     """
     args = parse_args()
+    if args.config_file:
+        zuliprc_path = args.config_file
+    else:
+        zuliprc_path = '~/zuliprc'
```

Comment:
This should be `~/.zuliprc`, but preferrably `$XDG_CONFIG_HOME/zulip/config.ini`, or whatever is the appropriate file extension for the used format.

`$XDG_CONFIG_HOME` should be expected to be empty, in which case it defaults to `$HOME/.config`.